externs/browser/w3c_xml.js

```diff
@@ -400,6 +400,36 @@ XMLHttpRequest.prototype.onreadystatechange;
 XMLHttpRequest.prototype.onerror;
 
 /**
+ * @type {number}
+ * @see https://www.w3.org/TR/XMLHttpRequest/#states
+ */
+XMLHttpRequest.UNSENT = 0;
```

Comment:
Assignments in externs are no-ops, so please remove them and only keep the type declaration.